src/common/tokenMintKeys.ts

```diff
@@ -1,6 +1,7 @@
 import { NATIVE_MINT } from "@solana/spl-token"
 
 export const mainTokenMintKeys = {
+  //wSOL's mint address
   WRAPPED_SOL: NATIVE_MINT,
```

Comment:
Keep it simple, devs are familiar with NATIVE_MINT
```suggestion
  /** Wrapped SOL's mint address */
  NATIVE_MINT,
```